app/tests/content/test_badge_integration.py

```diff
@@ -0,0 +1,160 @@
+from datetime import timedelta
+
+from rest_framework import status
+
+import pytest
+
+from app.content.factories.badge_factory import BadgeFactory
+from app.util.test_utils import get_api_client
+from app.util.utils import now
+
+API_BADGE_BASE_URL = "/badges/"
+
+
+def _get_badge_leaderboard_url():
+    return f"{API_BADGE_BASE_URL}leaderboard/"
+
+
+def _get_badge_specific_leaderboard_url(badge):
+    return f"{API_BADGE_BASE_URL}{badge.id}/leaderboard/"
+
+
+def _get_user_badge_url():
```

Comment:
Du henter ut badges i flertall ;)
```suggestion
def _get_user_badges_url():
```